programs/loader-v4/src/lib.rs

```diff
@@ -146,8 +146,16 @@ fn execute<'a, 'b: 'a>(
 ) -> Result<(), Box<dyn std::error::Error>> {
     // We dropped the lifetime tracking in the Executor by setting it to 'static,
     // thus we need to reintroduce the correct lifetime of InvokeContext here again.
-    let executable =
-        unsafe { std::mem::transmute::<_, &'a Executable<InvokeContext<'b>>>(executable) };
+    let executable = unsafe {
+        std::mem::transmute::<
```

Comment:
well, both of type params are identical in source code wise... That's not helpful for human while mitigating unexpected type inference change as motivated by the new lint...